WebRoot/js/zimbraAdmin/common/ZaApp.js

```diff
@@ -95,6 +95,57 @@ ZaApp.checkMyRight = function(targetType,targetBy,targetVal,right,attrs) {
 	
 }
 
+ZaApp.toggleLoadingStatusBar = function (instance, show, label) {
```

Comment:
Since it's used as a global function, it's easier to call ZaApp.toggleLoadingStatusBar than ZaApp.getInstance().toggleLoadingStatusBar. In this case, the instance parameter passed is that one of the current view (for this particular ticket, it's ZaNewVolumeXWizard).

Reply:
okay